desktop/assets/auctions.coffee

```diff
@@ -6,10 +6,12 @@ routes =
   /auctions/reminders
   ''': require('../apps/auctions/client/index.coffee').init
 
+  # TODO: refactor to check paths in order.
+  # Discussion here: https://github.com/artsy/force/pull/1126
   '''
   /sale/.*
-  /auction/.*
-  ''': require('../apps/auction/client/index.coffee').init
+  ^/auction/[^/]+/?$
+  ''': -> require('../apps/auction2/client/index.js').default
```

Comment:
I think you don't want the arrow here—that's wrapping the code in an anonymous function and not actually calling the `default` function.